packages/language-server/src/__test__/completion.test.ts

```diff
@@ -208,6 +208,28 @@ suite('Completions', function () {
         },
       })
     })
+
+    test('Diagnoses block type suggestions for view preview', () => {
+      assertCompletion({
+        schema: /* Prisma */ `
```

Comment:
Adding the generator for the preview feature flag is not needed actually.
You can set it like 
```
previewFeatures: ['view'],
```